src/world.rs

```diff
@@ -130,6 +174,19 @@ impl World {
         }
     }
 
+    /// Remove components for a given [Entity].
+    ///
+    /// # Example
+    ///
+    /// ```
+    /// # use rs_ecs::*;
+    /// let mut world = World::new();
+    ///
+    /// let entity = world.alloc();
+    /// world.insert(entity, (42_u32, true));
+    ///
+    /// world.remove::<(u32, )>(entity);
```

Comment:
I think Rustfmt would remove that trailing space after the comma? As with insert it is probably helpful to show both single component and component bundle variants in the example? (I understand the two methods together show everything but I think it is nice if each method documents those pitfalls on its own.)

Reply:
(This would also be a strong test by `unwrap`ing the result of the call.)

Reply:
Completely agreed.